website/addons/dryad/model.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+
+from modularodm import fields
+
+from website.addons.dryad import utils as dryad_utils
+from website.addons.base import AddonUserSettingsBase, AddonNodeSettingsBase
+
+
+class AddonDryadUserSettings(AddonUserSettingsBase):
+    """
+     Stub in place for Dryad addon v2 upgrade.
+
+     As migrations are a little tricky, this stub is being left in since a user
+     settings class will be needed in the future.
+    """
+    has_auth = True
+
+
+class AddonDryadNodeSettings(AddonNodeSettingsBase):
+    """
+        A Dryad node is a collection of packages. Each package is specified by a DOI, and the title is saved automatically
+    """
+    dryad_package_doi = fields.StringField()
+    has_auth = True
+    provider_name = 'dryad'
+    connection = dryad_utils.DryadRepository()
```

Comment:
It's somewhat of an anti-pattern to store initialized class-level objects. This really only matters when they store state, which this currently appears to not do. If it's expected that this might change with version 2 of this feature, consider instead being lazy:

```python
_api = None

@property
def api(self)
    if not self._api:
        self._api = dryad_utils.DryadRepository()
    return self._api
```